install.py

```diff
@@ -17,15 +17,16 @@
 
 DOWNLOAD_URL_FORMAT = 'https://github.com/grow/grow/releases/download/{version}/{name}'
 RELEASES_API = 'https://api.github.com/repos/grow/grow/releases'
-ALIAS_FILES_LINUX = ['.bash_aliases', '.bash_profile', '.profile', '.bashrc']
-ALIAS_FILES_MAC = ['.bash_profile']
-ALIAS_FILES_DEFAULT = '.bashrc'
-ALIAS_RE = re.compile(r'^alias grow\=([\'"])(.*)\1$', re.MULTILINE)
+RC_FILES_LINUX = ['.bash_profile', '.profile', '.bashrc']
```

Comment:
I think the fallback order should be `['.bashrc', '.bash_profile', '.profile']`, and should be the same for both Linux and Mac. We might also want to look into adding `zsh` config locations here as well since the next major version of Mac will use `zsh` as default.

Reply:
Updated the ordering and added the zsh config file as one of the places to search.